src/internet_identity/tests/integration/v2_api/authn_method_test_helpers.rs

```diff
@@ -26,3 +30,20 @@ pub fn test_authn_method() -> AuthnMethodData {
         last_authentication: None,
     }
 }
+
+pub fn create_identity_with_authn_method(
+    env: &StateMachine,
+    canister_id: CanisterId,
+    authn_method: &AuthnMethodData,
```

Comment:
Why is this now passed by reference while before it wasn't?

Reply:
Having the function only borrow the argument allows tests to use it multiple times. This moves the call to `clone` into the function making the tests less cluttered.

Reply:
None of the tests seem to be using the argument multiple times, nor calling clone. Am I missing something?

Is this change related to the moving the helper or orthogonal?

Is it needed/used by your follow-up/main PR?

Reply:
> None of the tests seem to be using the argument multiple times, nor calling clone. Am I missing something?

I think so? There is a line diff in `src/internet_identity/tests/integration/v2_api/authn_method_add.rs` that changes
```
create_identity_with_authn_method(&env, canister_id, authn_method.clone());
```

to 

```
let identity_number = create_identity_with_authn_method(&env, canister_id, &authn_method);
```
Similarly in the upcoming changeset.

> Is this change related to the moving the helper or orthogonal?

It does not directly have anything to do with moving, but goes in the same vein as it also makes the helper more useful (imho).

Reply:
Ah, yes, I missed it.

It does mean that clone is now called more often than before, right? Does that go against the spirit or Rust? Anyway it's fine.